packages/turbo-codemod/src/transforms/migrate-dot-env.ts

```diff
@@ -0,0 +1,103 @@
+import path from "node:path";
+import { readJsonSync, existsSync } from "fs-extra";
+import { type PackageJson, getTurboConfigs } from "@turbo/utils";
+import type { Schema as TurboJsonSchema } from "@turbo/types";
+import type { Transformer, TransformerArgs } from "../types";
+import { getTransformerHelpers } from "../utils/getTransformerHelpers";
+import type { TransformerResults } from "../runner";
+
+// transformer details
+const TRANSFORMER = "migrate-dot-env";
+const DESCRIPTION = 'Migrate the "dotEnv" entries to "inputs" in `turbo.json`';
+const INTRODUCED_IN = "2.0.0";
+
+function migrateConfig(config: TurboJsonSchema) {
+  if ("globalDotEnv" in config) {
+    if (config.globalDotEnv) {
```

Comment:
why two checks?

Reply:
Because if the field is set to null, we don't want to iterate over the entries, but we do want to delete the field still